mix_helpers.exs

```diff
@@ -252,13 +252,30 @@ defmodule Mix.Appsignal.Helper do
   end
 
   defp download_options do
-    options = [
-      ssl_options:
-        [
-          verify: :verify_peer,
-          cacertfile: priv_path("cacert.pem")
-        ] ++ tls_options() ++ customize_hostname_check_or_verify_fun()
-    ]
+    default_cacert_file_path = priv_path("cacert.pem")
+
+    options =
+      case check_cacert_access(default_cacert_file_path) do
+        :ok ->
+          [
+            ssl_options:
+              [
+                verify: :verify_peer,
+                cacertfile: default_cacert_file_path
+              ] ++ tls_options() ++ customize_hostname_check_or_verify_fun()
+          ]
+
+        {:error, message} ->
+          Logger.warn(
+            "The cacert file path: #{default_cacert_file_path} is not accessible. " <>
+              "Reason: #{inspect(message)}. " <>
+              "Using system defaults instead."
+          )
+
+          [
+            cacertfile: :certifi.cacertfile()
+          ]
```

Comment:
I don't think you can just pass `cacertfile` outside of the `ssl_options` object. If you do pass the `ssl_options`, you've got to pass the whole thing, like above, because passing it overrides all its default values (see the article I linked earlier)

Reply:
true!

Reply:
Changed it a bit so we always pass valid ssl_options